splunk_add_on_ucc_framework/global_config.py

```diff
@@ -113,7 +113,7 @@ def alerts(self) -> List[Dict[str, Any]]:
         return self._content.get("alerts", [])
 
     @property
-    def meta(self) -> Dict[str, str]:
+    def meta(self) -> Dict[str, Any]:
```

Comment:
In the updated code, would it be `Dict[str, Any[str, bool]]` or something else?

Reply:
You are partly right, it would be `Dict[str, Union[str, bool, <OS-dependent configuration>]]` so I decided just to go with `Dict[str, Any]`.